lib/compare-results/index.js

```diff
@@ -21,6 +21,12 @@ function throwMasterArtifactOutOfSyncError() {
     `);
 }
 
+function getfileNameFromTitle(title) {
```

Comment:
I'm trying to get the consistent file name from this title
`not ok 1 test/language/expressions/arrow-function/prototype-rules.js default # (expected success, got runtime error)
  ---`